sympy/printing/tests/test_llvmjit.py

```diff
@@ -0,0 +1,45 @@
+
+from sympy.external import import_module
+
+
+if import_module('llvmlite'):
+    import sympy.printing.llvmjitcode as g
+else:
+    disabled = True
```

Comment:
I see. That seems like a bad design. So, if you use py.test you can't use a variable called `disabled` except for signaling py.test to run? I haven't found anything in the py.test docs about this yet.